CHANGES_NEXT_RELEASE

```diff
@@ -11,3 +11,4 @@
     -  Fix or disable eslint errors
 - Upgrade logops dep from 2.1.0 to 2.1.2 due to colors dependency corruption
 - Add: graceful shutdown listening to SIGTERM and SIGHUP signals (#576)
+- Fixed undefined value of reconnectTries and reconnectInterval
```

Comment:
```suggestion
```

As this PR is a fix of the already existing:

```
- Add config (reconnectTries, reconnectInterval) to allow mongo driver reconnect (#559)
```

Reply:
@fgalan , Done in commit https://github.com/telefonicaid/fiware-sth-comet/pull/581/commits/72bd8432cbb0226a6c27da943518f08f94c9e21f